docs/source/reference/dataset_structure.rst

```diff
@@ -36,7 +36,16 @@ The corresponding class of dataset is :class:`~tensorbay.dataset.dataset.Dataset
 *******
 
 Notes contains the basic information of a dataset, such as the time continuity of the data inside the
-dataset.
+dataset. If a dataset is continuous, ``notes.is_continuous`` should be set as ``True``.
+
+.. code:: python
+
+   dataset.notes.is_continuous = True
+
+For example, notes is used in `LISA Traffic Light Dataloader`_ to specify `LISA Traffic Light Dataset`_ as a continuous dataset.
+
+.. _LISA Traffic Light Dataloader: https://github.com/Graviti-AI/tensorbay-python-sdk/blob/main/tensorbay/opendataset/LISATrafficLight/loader.py
+.. _LISA Traffic Light Dataset: https://gas.graviti.cn/dataset/hello-dataset/LISATrafficLight
```

Comment:
These are descriptions to the `Notes` class or the `notes` attribute of `dataset`, which should not be presented in the dataset structure section. If these descriptions have to be placed here, place them after the current L50.